configuration.md

```diff
@@ -359,7 +359,7 @@ The supported configuration options:
 
 ### `network`
 
-Specify networking options. For example:
+Specify [networking](networking/README.md) options. For example:
```

Comment:
Maybe remove the option definitions below the example as they are (or at least should be)  documented in the deeper pages.

Reply:
Is it now better?